src/library/components/Point/Point.tsx

```diff
@@ -2,36 +2,30 @@ import React, { FC } from 'react'
 import { Text, StyleSheet, View } from 'react-native'
 
 import { fonts } from 'res/fonts'
-import { lightenDarkenColor, colors, Color } from 'res/colors'
+import { colors } from 'res/colors'
 import { useT } from 'res/i18n'
 
 export interface PointProps {
-  points: number
-  shape?: 'circle' | 'rectangle'
-  backgroundColor: Color | string
+  points: number | string
 }
 
-export const Point: FC<PointProps> = ({
-  points,
-  backgroundColor,
-  shape = 'circle',
-}) => {
+export const Point: FC<PointProps> = ({ points }) => {
   const { t } = useT()
 
+  const pointsNumber = Number(points)
```

Comment:
Because the props points can be a string (1-1) or a number.
Idk how to improve it.

'1-1' => should display `1- 1 points`
'1' => should display `1 point`
'2' => should display `2 points`
3 => should display `3 points`

Reply:
I updated the code to be more simple. Can you take a look?